.github/workflows/export_category.yml

```diff
@@ -1,7 +1,9 @@
-name: Export Category
+name: Export Category 
 
 on:
   push:
+    paths:
+      - ".github/workflows/export_category.yml"
```

Comment:
This is where the code runs the export when a the yml file is changed as supposed to on push

Reply:
yep